go.mod

```diff
@@ -164,4 +164,4 @@ replace github.com/ethereum/go-ethereum => github.com/crypto-org-chain/go-ethere
 // TODO: remove when ibc-go and ethermint upgrades cosmos-sdk
 replace github.com/cosmos/cosmos-sdk => github.com/cosmos/cosmos-sdk v0.44.2
 
-replace github.com/tharsis/ethermint => github.com/crypto-org-chain/ethermint v0.7.2-cronos-4
+replace github.com/tharsis/ethermint => github.com/yihuang/ethermint v0.6.1-0.20211214073201-9fe3fe860947
```

Comment:
TODO, update after https://github.com/crypto-org-chain/ethermint/pull/35 merged and new tag created.